theme/less/_navigation.less

```diff
@@ -1,25 +1,133 @@
-nav
+// The main navigation menu
+
+#main-navigation 
 {
-  ul
+  background: linear-gradient(#fcfcfc, #ececec);
+}
+
+#main-navigation > ul
+{
+  display: flex;
+  padding: 0;
+  border: #F5F5F5 solid  1px;
+  border-radius: 5px;
+  box-shadow: 0 1px 1px rgba(0, 0, 0, 0.2);
+}
+
+#main-navigation > ul li
+{ 
+  display: flex;
+  justify-content:space-between;
+  position: relative;
+  list-style: none;
+  border-right: 1px solid #e3e3e3;   
+}
+
+#main-navigation ul ul li
+{
+  min-width: 150px;
+}
+
+#main-navigation > ul > li:hover
+{
+  background-color: #f0f0f0;
+}
+
+#main-navigation ul ul li:hover, ul ul a:hover
+{
+  background-color: #ffffff;
+}
+
+#main-navigation ul a
+{
+  padding: 15px;
+  flex-grow: 2;
```

Comment:
Detail: I assume this left since the times arrow element was sibling of anchor tag, so you have specified this to push the content all the way to the end.

Reply:
Basically we can replace it by changing  padding (increasing padding-right component) of `#main-navigation ul a`, because there is a gap between 2 flex items (not covered by anchor selectors). But in that case we don't know exact size of the gap, flex-grow in this case is what we need I think, because it covers the unused space between flex items. 
By the way we can remove  `justify-content:space-between;` because this taking no effect on positioning flex items and is overwritten with properties of `:after` selector as it is positioned absolutely.

Reply:
I still don't understand, why you need `flex-grow` here?

All `#main-navigation ul a` item siblings as far as I can see are positioned absolutely, so there seem to be no item you need to push by `flex-grow` element. Am I missing something?

> By the way we can remove justify-content:space-between;

Sounds good, please remove all unused items, I may have missed that, but if you tested and see no effect please go ahead.